tools/pyboard.py

```diff
@@ -370,6 +370,54 @@ def exec_raw_no_follow(self, command):
         if not data.endswith(b">"):
             raise PyboardError("could not enter raw repl")
 
+        t0 = time.time_ns()
+
+        if True:
```

Comment:
Should this `if True` be removed too? (I'm guessing it was another development-thing where you could easily change it to`if False` to skip running any of the newly-added code?)

Reply:
yes, also to be removed!